docs/examples/viz_parametric_surfaces.py

```diff
@@ -0,0 +1,180 @@
+###############################################################################
+# Importing necessary modules
+
+from fury import window, actor, ui, colormap
+import numpy as np
+
+###############################################################################
+# Function that returns the parametric surface specified by the user.
+
+
+def parametric_func(name, colors=(1, 1, 1), coords=[0, 0, 0],
```

Comment:
- call it just `parametric`
- This function should go in `actor` module. Also, it will be good to add a test for this function
- `centers` instead of `coords`
- `ntype` instead of `name`
- It should handle many centers and many `ntype` at the same time. I would recommend to look at the other actors like sphere, square, etc... You can create in one call, multiple objects.